src/pages/link-app/index.tsx

```diff
@@ -0,0 +1,184 @@
+import {
+  // AspectRatio,
+  Button,
+  Container,
+  createStyles,
+  Flex,
+  Grid,
+  Stack,
+  Text,
+  Title,
+  Image,
+} from '@mantine/core';
+// import { YoutubeEmbed } from '~/components/YoutubeEmbed/YoutubeEmbed';
+import { containerQuery } from '~/utils/mantine-css-helpers';
+import { useCurrentUser } from '~/hooks/useCurrentUser';
+import { NextLink } from '@mantine/next';
+import Lottie from 'react-lottie';
+import * as linkAnimation from '~/utils/lotties/link-animation.json';
+
+export default function LinkApp() {
+  const { classes } = useStyles();
+  const currentUser = useCurrentUser();
+  const isMember = currentUser?.isMember;
+  const buttonData = {
+    text: isMember ? 'Download the Link App' : 'Become a member',
+    href: isMember ? 'https://github.com/civitai/civitai-link-desktop/releases/latest' : '/pricing',
+  };
+
+  return (
+    <Container>
+      <Flex
```

Comment:
Would be great to add meta tags for better SEO probably. You can look for examples in the codebase by searching `<Meta`

Reply:
Good call.